rofeeah_ayeni/alpha_challenges/the_basics.md

```diff
@@ -108,14 +108,14 @@ What is the index value of "Michelangelo"?
 
 **Answer 8A:**
 ```ruby
-# write your code here
+# puts jurassic_park['Triceratops']
```

Comment:
Once you've added your code, you can uncomment the line (this is true for line 118, too).